packages/backend/src/payment-method/ilp/connector/core/middleware/error-handler.ts

```diff
@@ -11,10 +11,17 @@ export function createIncomingErrorHandlerMiddleware(
   serverAddress: string
 ): ILPMiddleware {
   return async (ctx: ILPContext, next: () => Promise<void>): Promise<void> => {
+    const stopTimer = ctx.services.telemetry.startTimer(
+      'create_incoming_error_handler_middleware_next',
+      {
+        callName: 'createIncomingErrorHandlerMiddleware:next'
+      }
+    )
     try {
       await next()
       if (!ctx.response.rawReply) {
         ctx.services.logger.error('handler did not return a valid value.')
+        stopTimer()
```

Comment:
can use a `finally` here to just have to do `stopTimer()` once

Reply:
Done.